database/seeds/development/11_access_role.seed.js

```diff
@@ -0,0 +1,10 @@
+exports.seed = (knex) =>
```

Comment:
Lol, I have this in my pr. I'm gunna have to rewrite all my test.

Reply:
Should just need to do find/replace, no?